bots/task/testmap.py

```diff
@@ -17,97 +17,98 @@
 
 REPO_BRANCH_CONTEXT = {
     'cockpit-project/cockpit': {
-        'master': ['avocado/fedora', 'container/bastion',
-            'selenium/firefox', 'selenium/chrome', 'selenium/edge',
-            'verify/debian-stable', 'verify/debian-testing',
-            'verify/ubuntu-1804', 'verify/ubuntu-stable',
-            'verify/fedora-30', 'verify/fedora-atomic',
-            'verify/rhel-8-0-distropkg', 'verify/rhel-8-1',
+        'master': ['fedora-30/avocado', 'fedora-30/container-bastion',
+            'fedora-30/selenium-firefox', 'fedora-30/selenium-chrome', 'fedora-30/selenium-edge',
+            'debian-stable', 'debian-testing',
+            'ubuntu-1804', 'ubuntu-stable',
+            'fedora-30', 'fedora-atomic',
+            'rhel-8-0-distropkg', 'rhel-8-1',
         ],
-        'rhel-7.6': ['avocado/fedora', 'container/kubernetes', 'container/bastion',
-            'selenium/firefox', 'selenium/chrome', 'verify/rhel-7-6',
+        'rhel-7.6': ['fedora-30/avocado', 'fedora-30/container-kubernetes', 'fedora-30/container-bastion',
+            'fedora-30/selenium-firefox', 'fedora-30/selenium-chrome', 'rhel-7-6',
         ],
-        'rhel-7.7': ['avocado/fedora', 'container/kubernetes', 'container/bastion',
-            'selenium/firefox', 'selenium/chrome', 'verify/rhel-7-7',
+        'rhel-7.7': ['fedora-30/avocado-fedora', 'fedora-30/container-kubernetes',
```

Comment:
containers, avocado and selenium are always F30 no matter which branch, right?

Reply:
I'd rather not make this assumption. We regularly update the image for these, and then also (have to) update the stable branches to keep up. But the point of this is to be more flexible, and always specify the OS. What's the background of your question?